### DISCUSSION TITLE
Hooks documentation

### DISCUSSION THREAD

**Discussion by Loosetooth** [reduxjs/react-redux#2120] [General]
I was recently trying to find documentation about support for hooks.
However, I noticed a few things 'hiding' the hooks documentation:

1. Hooks are under the 'API reference' tab, even though they are seemingly strongly recommended as a default approach:
![image](https://github.com/reduxjs/react-redux/assets/66420755/ac1bd665-285f-4cf5-934c-04bd415ab7d0)
2. Searching for 'hooks' gives no results: 
![image](https://github.com/reduxjs/react-redux/assets/66420755/4d37f0b0-d8d4-4627-8455-054e426fd142)

I think that this hides the hooks from a lot of users, even if they are actively trying to search for it.
What do you think? Is it worth moving the hooks to some place more noticeable and fix the search?

**Comment by markerikson** [comment]
That page _is_ an "API Reference" page, so it's located in the correct spot.

There may be some issues with our Algolia search integration that we need to address.
